thirdeye/thirdeye-pinot/src/main/resources/migrations.xml

```diff
@@ -553,4 +552,27 @@
     </createTable>
   </changeSet>
 
+   <changeSet id="21" author="npawar">
+  <!-- WebappConfigs -->
+    <createTable tableName="webapp_configs">
+      <column name="id" type="bigint" autoIncrement="true">
+        <constraints primaryKey="true" nullable="false"/>
+      </column>
+      <column name="config_id" type="bigint">
+        <constraints nullable="false"/>
+      </column>
+      <column name="collection" type="varchar(127)">
+        <constraints nullable="false"/>
+      </column>
+      <column name="config_type" type="varchar(127)">
```

Comment:
may be you can omit 'config' and keep it just type. As this is in webapp_config table, type means webapp_config_type.